apps/server/test/lexical/server/code_intelligence/completion_test.exs

```diff
@@ -102,11 +102,24 @@ defmodule Lexical.Server.CodeIntelligence.CompletionTest do
         name: "#{name}-callback",
         origin: full_name,
         argument_names: [],
-        metadata: %{}
+        metadata: %{},
+        arity: 0
       },
       %Candidate.Exception{name: "#{name}-exception", full_name: full_name},
-      %Candidate.Function{name: "my_func", origin: full_name, argument_names: [], metadata: %{}},
-      %Candidate.Macro{name: "my_macro", origin: full_name, argument_names: [], metadata: %{}},
+      %Candidate.Function{
+        name: "my_func",
+        origin: full_name,
+        argument_names: [],
+        metadata: %{},
+        arity: 0
+      },
+      %Candidate.Macro{
+        name: "my_macro",
+        origin: full_name,
+        argument_names: [],
+        metadata: %{},
+        arity: 0
+      },
```

Comment:
I added an arity of 0 to the mocks in this part (instead of adjusting the sort_text function to handle the arity value of nil) because I thought that was the realistic case. But I'm not super on that 😕 .

Reply:
seems reasonable